app/models/dependency.rb

```diff
@@ -12,8 +12,13 @@ class Dependency < ActiveRecord::Base
 
   attr_accessor :gem_dependency
 
+  def self.unresolved(rubygem)
+    where(:unresolved_name => nil, :rubygem_id => rubygem.id)
```

Comment:
can you make this use new Hash syntax? 
If you are touching this lines already, I think is a good idea to update it to new syntax.